src/btree/bt_io.c

```diff
@@ -365,6 +366,12 @@ __wt_bt_write(WT_SESSION_IMPL *session, WT_ITEM *buf,
 	WT_STAT_FAST_DATA_INCR(session, cache_write);
 	WT_STAT_FAST_CONN_INCRV(session, cache_bytes_write, dsk->mem_size);
 	WT_STAT_FAST_DATA_INCRV(session, cache_bytes_write, dsk->mem_size);
+	if (checkpoint_io) {
```

Comment:
@keithbostic This is useful and I definitely want to keep it, but it can be misleading if the application has compression enabled. I know it's a layering violation to figure this out in the block manager, but it's the only place we know the actual disk-block write size (unless it's hidden in the dsk structure) at the moment.